arbitrage/arbitrage.py

```diff
@@ -0,0 +1,18 @@
+class Query (object):
```

Comment:
This class should take in an API object at some point, possibly constructor. This will make it easy to test because we can fake the API. 

Look up dependency injection if you want to read about this design pattern.